CHANGELOG-3.0.md

```diff
@@ -0,0 +1,119 @@
+# Release notes for master
+
+[Documentation](https://docs.k8s.io/docs/home)
+
+# Changelog since v2.2.0
```

Comment:
minor nit: but it would have been nicer to first remove the godep compatibility path and then generate CHANGELOG document, so as it is captured in `CHANGELOG` document.

Reply:
It will be picked by the release-notes script when generating final release notes.